### PR TITLE
Add command completion file

### PR DESCRIPTION
**Pull request by suganyasuven**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/29679

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Comment by keizer619** [comment]
is above file copied into tools zip?

File: distribution/zip/jballerina/build.gradle [hunk 1]
```diff
@@ -118,9 +118,14 @@ CopySpec copyStaticSpec = copySpec {
...
-    from files('resources')
-    into(basePath + '/bre')
+CopySpec copySecuritySpec = copySpec {
+    from files('resources/security')
+    into(basePath + '/bre/security')
+}
+
+CopySpec copyCompletionSpec = copySpec {
+    from files('resources/command-completion')
+    into(basePath + '/')
```
Comment (keizer619) [inline distribution/zip/jballerina/build.gradle:128]: This is copied into pack root. Shall we copy that into a scripts director?

**Comment by suganyasuven** [comment]
> is above file copied into tools zip?



> is above file copied into tools zip?

Yes with this commit 6e1beaeb0e94bf092cc4e9290cc89814f7843b7f

File: distribution/zip/jballerina/build.gradle (hunk 1 above)
Comment (keizer619) [reply distribution/zip/jballerina/build.gradle:128]: since its a csv we can use existing resourced directory

**Review by keizer619 — APPROVED** [review]

**Pull request merged**